### PR TITLE
[docs] Selectors - Add info about string arguments to withText and withAttribute methods

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs

### CODE REVIEW

File: docs/articles/documentation/test-api/selecting-page-elements/selectors.md [hunk 1]
```diff
@@ -176,6 +176,9 @@ Method | Type | Description
 `withText(text)` | Selector | Creates a selector that filters a matching set by the specified text.
 `withText(re)` | Selector | Creates a selector that filters a matching set using the specified regular expression.
 
+The `withText(text)` method converts the string argument to a regular expression.
```
Comment (churkin) [inline docs/articles/documentation/test-api/selecting-page-elements/selectors.md]: Does it ignore cases?

Comment (helen-dikareva) [inline docs/articles/documentation/test-api/selecting-page-elements/selectors.md]: it's case sensitive

Comment (churkin) [inline docs/articles/documentation/test-api/selecting-page-elements/selectors.md]: @VasilyStrelyaev May be note about this?

Comment (AlexanderMoskovkin) [inline docs/articles/documentation/test-api/selecting-page-elements/selectors.md]: > The `withText(text)` method converts the string argument to a regular expression.

It's not true. We should say only that `withText(text)` it's not about an element's text content is strict equal to `text` but element's text content contains `text`

**Comment by VasilyStrelyaev** [comment]
@testcafe-build-bot \retest

**Pull request merged**